dashboard/app/models/script.rb

```diff
@@ -661,6 +661,39 @@ def link
     Rails.application.routes.url_helpers.script_path(self)
   end
 
+  def can_be_instructor?(user)
+    # If unit is in a unit group then decide based on unit group audience
+    return unit_group.can_be_instructor?(user) if unit_group
+
+    return false if user.student?
+    return true if user.permission?(UserPermission::CODE_INSTRUCTOR) || user.permission?(UserPermission::LEVELBUILDER)
```

Comment:
What's a code instructor? Also, just curious if verification status for teachers factors in to any of these permissions? Trying to keep straight what everything does 🙃

Reply:
A code instructor is a new permission that just allows us to designate someone has being able to be the instructor of any course. 

Verification of teachers/instructors is going to be dealt with here: https://github.com/code-dot-org/code-dot-org/pull/43375. It determines if instructors can see verified resources or answers.

Reply:
Does the "code" in `CODE_INSTRUCTOR` refer to "Code.org"?  If so, I wonder if renaming it `CODE_ORG_INSTRUCTOR`, `CODE_DOT_ORG_INSTRUCTOR` or possibly `CDO_INSTRUCTOR` would make it clearer.

Reply:
I can see how CODE_INSTRUCTOR could be confusing. @jamescodeorg @Erin007 do you have have a preference of the suggestion James made?

Reply:
I will follow up with an update to CODE_INSTRUCTOR name in another PR.